src/index.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Router from './App';
 import registerServiceWorker from './registerServiceWorker';
+import { Provider } from 'react-redux';
+import configureStore from './contact/store/store';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<Provider store = {configureStore()}> 
+		<Router /> 
+	</Provider>, document.getElementById('root'));
```

Comment:
There as a class defination: div.root in css, but I see an id as well here in this piece of code, Do you want explain why id and classes are used.

Reply:
class is for giving styles. ID is for selecting the root component in index file. We can also select on the basis of class.